Mathlib/Order/RelClasses.lean

```diff
@@ -137,6 +137,22 @@ theorem trans_trichotomous_right [IsTrans α r] [IsTrichotomous α r] {a b c : 
 
 theorem transitive_of_trans (r : α → α → Prop) [IsTrans α r] : Transitive r := IsTrans.trans
 
+/-- In a symmetric and transitive relation, replacing the left argument in a relation by a related
+argument given an equivalent relation. -/
+theorem rel_congr_left [IsSymm α r] [IsTrans α r] {a b c : α} (h : r a b) : r a c ↔ r b c :=
```

Comment:
The docstring reads like there are two relations. Can you clarify?

Reply:
I'm not really sure how to write this better. I think I'll just remove the comment, since this statement isn't particularly hard to read.